README.md

```diff
@@ -67,14 +67,14 @@ vastParser.parseVAST(vastXml)
 
 ### VASTTracker
 
-To track the execution of an ad use the **VASTTracker**:
+To track the execution of an ad, by calling [VAST tracking elements](https://iabtechlab.com/wp-content/uploads/2019/06/VAST_4.2_final_june26.pdf#page=28), use the **VASTTracker**:
```

Comment:
`To track the execution of an ad, create a **VASTTracker** instance and call the [VAST tracking elements](https://iabtechlab.com/wp-content/uploads/2019/06/VAST_4.2_final_june26.pdf#page=28)`
Semantically better in that order imo.